src/Tools/TestDiscoveryWorker/TestDiscoveryWorker.csproj

```diff
@@ -4,12 +4,14 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>$(NetRoslyn);net472</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Curious why was this added?

Reply:
The `Mono.Options` reference isn't signed hence this can't be either since it uses it.